code/game/objects/machinery/vending/marine_vending.dm

```diff
@@ -650,14 +702,12 @@
 		/obj/item/clothing/shoes/marine = 20,
 		/obj/item/clothing/under/marine/standard = 20,
 		/obj/item/clothing/under/marine/jaeger = 30,
+		/obj/item/clothing/suit/storage/marine/harness = 50,
```

Comment:
The harness, while yes not actually providing any armor whatsoever, fits into the 'armor' slot so to speak.

also, don't make it 50 of them. Keep it at 5.